src/main/java/Duke.java

```diff
@@ -1,10 +1,148 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    public static Task[] tasks = new Task[100];
+    public static int numTasks;
+
+    public static void printMessage(String message) {
+        String s = "     ";
+        String line = "    _____________________________________"
+                + "__________________________________\n";
+        System.out.println(line);
+        message = message.replace("\n", "\n" + s);
+        System.out.println(s + message);
+        System.out.println(line);
+    }
+
+    public static void printWelcomeMessage() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String welcomeMessage = "Hello from \n" + logo + "\n"
+                + "What can I do for you?";
+        printMessage(welcomeMessage);
+    }
+
+    public static void printGoodbyeMessage() {
+        printMessage("Bye. Hope to see you again soon!");
+    }
+
+    public static void listTasks() {
```

Comment:
it may be a bit neater to move methods like `listTasks, addTask, deleteTask` etc. into a separate class called `TaskList` maybe

Reply:
Thank you for your comments! As it turns out, this suggestion is also provided in `A-MoreOOP`, but I haven't reached there yet. I was still at Level-6. I will follow the task requirements now.